report/6.Conclusion/Perspectivation.tex

```diff
@@ -1 +1,102 @@
+	\item ability to hit a small moving target in the air at a low speed at a distance of 1 to 3 meters
+	\item ability to predict the next location of the object moving along a non-linear path, eg{.} a target that continuously moves back and forth on the vertical axis while moving linearly on the horizontal axis.
+	\item ability to predict the next location of the object moving solely along the horizontal axis with a constant speed
+\end{itemize}
+
+As mentioned in the must have conclusion, the robot is able to track a balloon falling at a distance of 3 meters.
+The robot is also able to track a large object, such as a balloon, being moved at a slow speed with unpredictable movement patterns.
+At a distance of 1 meters, an object with a size of at least 15 centimeters in diameter can be tracked with a good amount of precision.
+Being able to track smaller objects would require more precise motors as the imprecision of the motors causes some issues, as they can only move a full degree per step, meaning the movement of the laser will become more imprecise as the distance to the target increases.
+For example, at a distance of 30 centimeters from the laser, one degree movement will result in the laser moving 0.3 centimeters, however, at a distance of 2.5 meters, moving a single degree will result in the laser moving more than 4 centimeters.
+This limitation means that at a distance, hitting a small object is simply not viable due to the limitations of the motors.
+Likewise, the ability to predict the next location of an object was relinquished as it was not deemed necessary when not firing a projectile.
+Thus, 2 of the 4 \textit{should have} requirements have been fulfilled.
+
+\subsubsection{Could have and won't have}
+From the \textit{could have} section, the following requirements are formulated:
+\begin{itemize}
+	\item ability to calculate the distance to the target object
+	\item calculate the angle and power necessary to hit the target with a projectile
+\end{itemize}
+
+And in the \textit{won't have}:
+\begin{itemize}
+	\item ability to actually take down targets
+	\item ability to differentiate between the type of object that is being observed
+\end{itemize}
+
+None of these requirements have been fulfilled, as they are largely related to firing a projectile.
+
+\subsubsection{Conclusion}
+In conclusion, all of the must have requirements and half of the should have requirements have been fulfilled.
+This is enough to deem the project successful, as most of the unfulfilled requirements are related to firing a projectile, which was excluded from the scope of the project. 
+
+\subsection{Test cases}
+Along with the MoSCoW analysis, four test cases were presented: 
+\begin{itemize}
+    \item Dropping a balloon
+    \item Rolling a ball
+    \item Throwing a ball
+    \item Shooting a balloon
+\end{itemize}
+
+The first case, dropping a ball, was already presented in the MoSCoW conclusion and deemed successful.
```

Comment:
Was already presented? Where? I guess it is meant to be: As part of the must have requirements, men guess det lige skal specificeres.